initializr-generator-spring/src/test/resources/project/gradle/annotation-processor-dependency-build.gradle.gen

```diff
@@ -6,7 +6,9 @@ plugins {
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
+java {
```

Comment:
That's an odd place to put this. Now that it isn't an attribute anymore, it should go after the dependencies IMO.

Reply:
Thanks, I guess I am biased with how things work with Maven. It's all good keeping it there and yes with a blank line.